source/guides/exchanges/exchange-integration.rst

```diff
@@ -0,0 +1,353 @@
+
+Integration overview
+********************
+
+There are many ways to design an exchange. This guide is based on how to support ``XYM`` deposits and withdrawals in an exchange that follows a **central wallet approach**.
+
+Please note that this design is not particularly recommend over others. However, its **simplified architecture** is a good showcase for Symbol's set of features involved in integrating with an Exchange. A different approach, for example, would be to use a different wallet for each user.
+
+.. figure:: /resources/images/diagrams/exchange-integration-overview.png
+   :align: center
+   :width: 100%
+   :target: /_images/exchange-integration-overview.png
+
+   **Fig. 1**: General design diagram of the central wallet approach.
+
+The main components of this architecture are described next.
+
+Components
+==========
+
+Central wallet
+--------------
+
+The exchange owns a Symbol account where all the user's deposits and withdrawals occur. This account only has the necessary amount of XYM for daily use (withdrawals and deposits), since it is the account most exposed to attacks.
+
+To increase the account security, we highly recommend to set up the wallet using Symbol's :doc:`multisig feature <../../concepts/multisig-account>`. Withdrawals can be automated using bots to ensure wallet security with **two-factor authentication**.
+
+Cold wallet
+-----------
+
+Cold wallet(s) hold a certain threshold for the pool of XYM. These accounts should be created and remain in a setup with no internet connection. Transactions issued from cold wallets must be signed offline and announced to the network using another device. It is advisable as well that cold wallets are set up with :doc:`multisig accounts <../../concepts/multisig-account>`, this time controlled by people.
+
+Unique User ID
+--------------
+
+Each user is identified by a unique identifier (UUID) on the exchange's database. A user will deposit to the central wallet with their UUID attached as
+the :doc:`message <../../concepts/transfer-transaction>` of the transaction (called sometimes the **memo**). The UUID is only shown
+in the user's dashboard during the deposit confirmation.
+
+One of the drawbacks of this design is that many users are not used to having a message attached to their transactions. If they forget to attach the UUID or attach a wrong UUID, it will lead to receiving lots of support tickets concerning "lost funds".
+
+.. caution::
+
+   Symbol's :doc:`Transfer transactions <../../concepts/transfer-transaction>` can hold an arbitrary message up to 1023 bytes long but **the first byte is treated specially by the** :doc:`Symbol SDK <../../sdk>`.
+
+   This can be a source of confusion because the receiver of a transaction does not know if the message was generated by the Symbol SDK or otherwise (for example accessing the :doc:`REST gateway <../../api>`), so it does not know if the first byte must be treated specially or not.
```

Comment:
The problem is the confusion between the sender of a tx and the receiver, right? The exchange may or may not use the SDK, but it will never know if the sender of the tx used it, therefore it is safer to always ignore the first byte.
I thought this was what we agreed when we wrote this:
https://nem-software.atlassian.net/wiki/spaces/SES/pages/600605077/Memo+restrictions